File: UNIDAD 1/00_SOA_SIN_BDD/00. SOAP_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/Factura.kt
package ec.edu.monster.modelo

import java.io.Serializable
import java.math.BigDecimal

data class Factura(
    val idFactura: Int,
    val numeroFactura: String,
    val fechaEmision: String,
    val idEmpleado: Int,
    val idCliente: Int,
    val idMetodoPago: Int,
    val subtotal: BigDecimal,
    val descuento: BigDecimal,
    val iva: BigDecimal,
    val total: BigDecimal,
    val empleado: Empleado? = null,
    val cliente: Cliente? = null,
    val metodoPago: MetodoPago? = null,
    val detallesFactura: List<DetalleFactura>? = emptyList()
) : Serializable

File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/02. CLIESC/VIAJECITOS_REST_CLIESC_GR10/VIAJECITOS_REST_CLIESC_GR10/Modelos/Factura.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VIAJECITOS_REST_CLIESC_GR10.Modelos
{
    public class Factura
    {
        public int IdFactura { get; set; }
        public string NumeroFactura { get; set; }
        public DateTime FechaEmision { get; set; }
        public int IdEmpleado { get; set; }
        public int IdCliente { get; set; }
        public int IdMetodoPago { get; set; }
        public decimal Subtotal { get; set; }
        public decimal Descuento { get; set; }
        public decimal Iva { get; set; }
        public decimal Total { get; set; }
        public Empleado Empleado { get; set; }
        public Cliente Cliente { get; set; }
        public MetodoPago MetodoPago { get; set; }
        public List<DetalleFactura> DetallesFactura { get; set; }
    }
}
